apps/python-sdk/firecrawl/v2/types.py

```
@@ -775,12 +775,63 @@
+    integration: Optional[str] = None
+
+class VectorSearchResult(BaseModel):
+    """A vector search result."""
+    id: str
+    url: str
+    title: Optional[str] = None
+    content: Optional[str] = None
+    similarity: float
+    metadata: Dict[str, Any]
+
+class VectorSearchTiming(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)